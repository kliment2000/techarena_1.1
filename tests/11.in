4
50 4 100 80
5
1 a 50
1 b 40
2 b 1
3 c 50
4 a 2
1
1 a
2
1 2 b b
3 4 c a
