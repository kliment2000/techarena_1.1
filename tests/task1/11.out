((3 (2 1a {2.b 1.b})) 4 {3.c 4.a}) 445.72
